test/extended/builds/gitauth.go

```diff
@@ -1,8 +1,11 @@
 package builds
 
-import (
-	"fmt"
+// THESE TESTS ARE DISABLED BECAUSE THE XIO.IO DNS HOOK WAS PROVING WAY TOO UNRELIABLE;
```

Comment:
shouting ? .... ah, cause of all caps ;-) .... sorry, i'm not that internet savvy 

and @bparees can appreciate my typo there :-)